documentation/src/bootstrap_node.md

```diff
@@ -0,0 +1,93 @@
+# Bootstrap node
+
+## Introduction
+
+The bootstrap node is the first node a new node contacts when it joins the
```

Comment:
Perhaps we could provide some additional links on how the bootstrap process works.  Our process should be the same as described here https://blog.ipfs.tech/2023-rust-libp2p-based-ipfs-bootstrap-node/#ipfs-public-dht-bootstrap-nodes